content/about/limits.md

```diff
@@ -1,44 +1,78 @@
+
+| Plan             | [CPU Limit](#cpu-execution-time-limit) | [Daily Request Limit](#daily-request-limit) | [Burst Rate Limit](#burst-rate-limit) |
+| ---------------- | --------------------------------------- | ------------------------------------------- | ------------------------------------- |
+| Free             | 10ms                                    | 100,000                                     | 1000 requests per 10 min              |
+| Unlimited (Paid) | 50ms                                    | none                                        | none                                  |
+
+See definitions for the types of limits and behavior below.
+Script size, number of scripts, subrequests, and available memory are not affected by plan type.
 
 ## Script Size
 
-A Workers script plus any [Asset Bindings](/tooling/api/bindings) can be up to 1MB in size after compression.
+A Workers script plus any [Asset Bindings](/tooling/api/bindings) can be up to **1MB** in size after compression.
 
 ## Number of Scripts
 
-Unless otherwise negotiated as a part of an enterprise level contract, all Workers accounts are limited to a maximum of 30 scripts at any given time.
+Unless otherwise negotiated as a part of an enterprise level contract, all Workers accounts are limited to a maximum of **30 scripts at any given time**.
 
-**Note:** app Workers scripts do not count towards this limit.
+**Note:** App Workers scripts do not count towards this limit.
 
-## Number of Requests Limit
+## Request Limits
 
 Unlimited (Paid) Workers scripts automatically scale onto thousands of Cloudflare edge servers around the world; there is no general limit to the number of requests per second Workers can handle.
 
 Cloudflare's abuse protection methods do not affect well-intentioned traffic. However, if you send many thousands of requests per second from a small number of client IP addresses, you can inadvertently trigger Cloudflare's abuse protection. If you expect to receive `1015` errors in response to traffic or expect your application to incur these errors, contact Cloudflare to increase your limit.
 
-Accounts using the Workers free tier are limited to a maximum of 100,000 requests per day, with a burst rate limit of 1000 requests per 10 minutes. This limit applies at the account level, meaning that requests on your workers.dev subdomain count toward the same limit as your zoned domains. Visitors who run into the rate limit will be served a Cloudflare 1015 error page, however if you are calling your script programmatically, you can detect the rate limit page and handle it yourself by looking for HTTP status code 429. Upgrading to a paid plan will automatically lift this limit.
+The burst rate and daily request limits apply at the account level, meaning that requests on your workers.dev subdomain count toward the same limit as your zones. Upgrade to a paid plan to automatically lift these limits.
```

Comment:
can we add a link to upgrade here if possible? want to make it as frictionless as possible for people to upgrade

Reply:
added a link to https://dash.cloudflare.com/?account=workers/plans